fern/tutorials/welcome/web3-tutorials-overview/community-tutorials.mdx

```diff
@@ -116,4 +116,4 @@ Tutorials
 
 Want to keep learning? Check out our [Road to Web3 series](/docs/welcome-to-the-road-to-web3/?a=18798bffa0) (10 part tutorial series) or our comprehensive [Alchemy University](https://university.alchemy.com/?a=18798bffa0) (it’s free!).
 
-If you'd like your web3 technical content to be included, send us an email at [\[email protected\]](/cdn-cgi/l/email-protection#88e9e5f8e4e1eef1c8e9e4ebe0ede5f1a6ebe7e5).
+If you'd like your web3 technical content to be included, send us an email at amplify@alchemy.com.
```

Comment:
Does anyone still check this email?

Reply:
Not sure but that's what we had in old docs!